src/Raven.Client/Documents/Conventions/DocumentConventions.cs

```diff
@@ -56,6 +56,7 @@ public sealed class DocumentConventions : Client.Conventions
             MaxContextSizeToKeep = new Size(PlatformDetails.Is32Bits == false ? 8 : 2, SizeUnit.Megabytes),
 #if NETCOREAPP3_1_OR_GREATER
             HttpPooledConnectionLifetime = TimeSpan.FromMinutes(19),
+            HttpVersion = System.Net.HttpVersion.Version20,
```

Comment:
you want to set the policy here?

Reply:
I saw that the problem was in the httpClient in the RequestExecutor, so I changed it accordingly.

Reply:
We already set it to http2 in ctor, no? 
https://github.com/LielNagar/ravendb/blob/01c45ace6541e2b0298265b99dece8f9dbdb0954/src/Raven.Client/Documents/Conventions/DocumentConventions.cs#L305

do we need to set it again here ?

Reply:
I think we need to set the `HttpVersionPolicy` here and then use it in the http client